### PR TITLE
Update CollectSamErrorMetrics.wdl

### PR DESCRIPTION
**Pull request by lfleming1022**
Adding more error metrics to collect

### CODE REVIEW

**Review by yueyaog — APPROVED** [review]

File: CollectSamError/CollectSamErrorMetrics.wdl [hunk 1]
```diff
@@ -51,7 +51,17 @@ task CollectSamErrorMetricsTask {
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
         ERROR_METRICS="ERROR:CYCLE" \
         ERROR_METRICS="ERROR:GC_CONTENT" \
-        ERROR_METRICS="ERROR"
+        ERROR_METRICS="ERROR" \
```
Comment (fleharty) [inline CollectSamError/CollectSamErrorMetrics.wdl:54]: Can you remove the MAPPING_QUALITY stratifier from all stratifiers.  Add the -Minimum mapping quality 60 option.

**Review by fleharty — CHANGES_REQUESTED** [review]

File: CollectSamError/CollectSamErrorMetrics.wdl [hunk 2]
```diff
@@ -43,15 +43,27 @@ task CollectSamErrorMetricsTask {
...
         java -Xms2000m -jar /usr/gitc/picard.jar CollectSamErrorMetrics I=~{input_bam} \
-        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} \
+        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} MQ=~{minMQ} \
         ERROR_METRICS=null \
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
         ERROR_METRICS="ERROR:CYCLE" \
         ERROR_METRICS="ERROR:GC_CONTENT" \
-        ERROR_METRICS="ERROR"
+        ERROR_METRICS="ERROR" \
+        ERROR_METRICS="ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEXT" \
```
Comment (fleharty) [inline CollectSamError/CollectSamErrorMetrics.wdl:57]: Let's remove this context:
ERROR_METRICS="ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEX

File: CollectSamError/CollectSamErrorMetrics.wdl [hunk 3]
```diff
@@ -43,15 +43,27 @@ task CollectSamErrorMetricsTask {
...
+        ERROR_METRICS="ERROR:CYCLE" \
+        ERROR_METRICS="ERROR:BASE_QUALITY" \
+        ERROR_METRICS="ERROR:ONE_BASE_PADDED_CONTEXT" \
+        ERROR_METRICS="ERROR:HOMOPOLYMER" \
+        ERROR_METRICS="OVERLAPPING_ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEXT" \
```
Comment (fleharty) [inline CollectSamError/CollectSamErrorMetrics.wdl:62]: Let's remove this context:
ERROR_METRICS="OVERLAPPING_ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEXT" \

File: CollectSamError/CollectSamErrorMetrics.wdl [hunk 4]
```diff
@@ -43,15 +43,27 @@ task CollectSamErrorMetricsTask {
...
-        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} \
+        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} MQ=~{minMQ} \
         ERROR_METRICS=null \
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
         ERROR_METRICS="ERROR:CYCLE" \
         ERROR_METRICS="ERROR:GC_CONTENT" \
-        ERROR_METRICS="ERROR"
+        ERROR_METRICS="ERROR" \
+        ERROR_METRICS="ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEXT" \
+        ERROR_METRICS="ERROR:CYCLE" \
```
Comment (fleharty) [inline CollectSamError/CollectSamErrorMetrics.wdl:58]: CYCLE is specified twice.

File: CollectSamError/CollectSamErrorMetrics.wdl (hunk 1 above)
Comment (fleharty) [reply CollectSamError/CollectSamErrorMetrics.wdl:56]: Also remove:
ERROR_METRICS="ERROR" \

**Review by fleharty — CHANGES_REQUESTED** [review]
A few more lines to remove.

File: CollectSamError/CollectSamErrorMetrics.wdl [hunk 5]
```diff
@@ -43,15 +43,23 @@ task CollectSamErrorMetricsTask {
...
         Int memory_gb = 16
+        Int minMQ = 60
+
     }
     command {
         java -Xms2000m -jar /usr/gitc/picard.jar CollectSamErrorMetrics I=~{input_bam} \
-        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} \
+        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} MQ=~{minMQ} \
         ERROR_METRICS=null \
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
```
Comment (fleharty) [inline CollectSamError/CollectSamErrorMetrics.wdl:53]: Can you add:
OVERLAPPING_ERROR:INSERT_LENGTH
OVERLAPPING_ERROR:GC_CONTENT

and add an option that we can expose to Terra to the probability with a default of 1.

**Review by fleharty — CHANGES_REQUESTED** [review]

**Review by fleharty — APPROVED** [review]
Thanks @laurafleming-1022 Looks great!

**Pull request merged**